Regression in version 3.3.1

**Describe the bug**

Regression on vwersion 3.3.1

**What is the current behavior?**

I get the following error:
```
     + 1 hidden asset
    Entrypoint undefined = desktop.html
    [0] ../html-webpack-plugin/lib/loader.js!/src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/desktop.html.ejs 15.7 KiB {0} [built]
    [2] ../webpack/buildin/global.js 177 bytes {0} [built]
    [3] ../webpack/buildin/module.js 511 bytes {0} [built]
    [4] /src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/image/favicon.ico 64 bytes {0} [built]
    [5] /src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/image/logo.png 61 bytes {0} [built]
    [6] /src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/image/background-layer-button.png 80 bytes {0} [built]
        + 1 hidden module
Child html-webpack-plugin for "iframe_api.html":
     3 assets
    Entrypoint undefined = iframe_api.html
    [0] ../html-webpack-plugin/lib/loader.js!/src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/iframe_api.html.ejs 4.62 KiB {0} [built]
    [2] ../webpack/buildin/global.js 177 bytes {0} [built]
    [3] ../webpack/buildin/module.js 511 bytes {0} [built]
    [4] /src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/image/favicon.ico 64 bytes {0} [built]
    [5] /src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/image/background-layer-button.png 80 bytes {0} [built]
        + 1 hidden module
Child html-webpack-plugin for "mobile.html":
     3 assets
    Entrypoint undefined = mobile.html
    [0] ../html-webpack-plugin/lib/loader.js!/src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/mobile.html.ejs 9.76 KiB {0} [built]
    [2] ../webpack/buildin/global.js 177 bytes {0} [built]
    [3] ../webpack/buildin/module.js 511 bytes {0} [built]
    [4] /src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/image/favicon.ico 64 bytes {0} [built]
    [5] /src/geoportal/testgeomapfish_geoportal/static-ngeo/js/apps/image/logo.png 61 bytes {0} [built]
    [6] ./contribs/gmf/src/icons/layers.svg 332 bytes {0} [built]
        + 1 hidden module
                            Thanks for using Webpack!
                 Please consider donating to our Open Collective
                        to help us maintain this package.
                 Donate: https://opencollective.com/webpack/donate
fs.js:114
    throw err;
    ^
Error: EPERM: operation not permitted, utime '/usr/lib/node_modules/webpack-cli/bin/opencollective.js'
    at utimesSync (fs.js:1108:3)
    at compilerCallback (/usr/lib/node_modules/webpack-cli/bin/cli.js:336:8)
    at compiler.run (/usr/lib/node_modules/webpack-cli/bin/cli.js:361:7)
    at finalCallback (/usr/lib/node_modules/webpack/lib/Compiler.js:220:39)
    at hooks.done.callAsync.err (/usr/lib/node_modules/webpack/lib/Compiler.js:269:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/usr/lib/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (/usr/lib/node_modules/webpack/node_modules/tapable/lib/Hook.js:154:20)
    at emitRecords.err (/usr/lib/node_modules/webpack/lib/Compiler.js:267:22)
    at Compiler.emitRecords (/usr/lib/node_modules/webpack/lib/Compiler.js:449:39)
    at emitAssets.err (/usr/lib/node_modules/webpack/lib/Compiler.js:261:10)
    at hooks.afterEmit.callAsync.err (/usr/lib/node_modules/webpack/lib/Compiler.js:435:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/usr/lib/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at AsyncSeriesHook.lazyCompileHook (/usr/lib/node_modules/webpack/node_modules/tapable/lib/Hook.js:154:20)
    at asyncLib.forEachLimit.err (/usr/lib/node_modules/webpack/lib/Compiler.js:432:27)
    at /usr/lib/node_modules/webpack/node_modules/neo-async/async.js:2813:7
    at done (/usr/lib/node_modules/webpack/node_modules/neo-async/async.js:3521:9)
    at /usr/lib/node_modules/webpack/node_modules/graceful-fs/graceful-fs.js:45:10
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)
CONST_Makefile:604: recipe for target '/build/apps.timestamp' failed
make: *** [/build/apps.timestamp] Error 1
```

**To Reproduce**
1. Install Webpack globally `npm install --global ...` (in my case in a Linux Docker container).
2. Build the project using webpack.


**Expected behavior**
Build success


**Please paste the results of `webpack-cli info` here, and mention other relevant information**

Full log: 
https://travis-ci.org/camptocamp/c2cgeoportal/builds/525892796#L4685

**Additional context**
Looks that's this line is in charge of this issue: 
https://github.com/webpack/webpack-cli/pull/755/files/55992a45a302977ef5e5075e81cc127e9fafdf07#diff-22d5d2b57c69d2e1ac054c4d0f744ff1R493

--- a/bin/cli.js
+++ b/bin/cli.js
@@ -326,14 +326,21 @@ For more information, see https://webpack.js.org/api/cli/.`);
 					const SIX_DAYS = 518400000;
 					const now = new Date();
 					if (now.getDay() === MONDAY) {
-						const { statSync, utimesSync } = require("fs");
+						const {
+							access,
+							constants,
+							statSync,
+							utimesSync,
+						} = require("fs");
 						const lastPrint = statSync(openCollectivePath).atime;
 						const lastPrintTS = new Date(lastPrint).getTime();
 						const timeSinceLastPrint = now.getTime() - lastPrintTS;
 						if (timeSinceLastPrint > SIX_DAYS) {
 							require(openCollectivePath);
 							// On windows we need to manually update the atime
-							utimesSync(openCollectivePath, now, now);
+							access(openCollectivePath, constants.W_OK, (e) => {
+								if (!e) utimesSync(openCollectivePath, now, now);
+							});
 						}
 					}
 				}

